CRM/Utils/REST.php

```diff
@@ -546,6 +546,14 @@ public static function ajaxJson() {
       );
       CRM_Utils_JSON::output($error);
     }
+    // FIXME: Shouldn't the X-Forwarded-Proto check be part of CRM_Utils_System::isSSL()?
```

Comment:
@ineffyble

Can you test if this would work replacing Lines 549 - 556 (new) just replacing with 

```php

 CRM_Utils_System::loadBootStrap(array(), FALSE, FALSE);
 CRM_Utils_System::redirectToSSL();

```

Can you see if that will still do the same job. The only thing i see is that L549-553 is duplicated in the redirectToSSL function